Babylon/commands/api/solution/update.py

```diff
@@ -23,7 +23,7 @@
 @timing_decorator
 @require_platform_key("api_url")
 @pass_azure_token("csm_api")
-@argument("solution_id", type=QueryType())
+@argument("solution_id", type=QueryType(), default="%deploy%solution_id")
```

Comment:
```suggestion
@argument("solution_id", type=QueryType())
```
Same for update I prefer to be explicit